packages/sanity/src/core/studio/components/navbar/appearance/appearance.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react'
+import {Box, Button, Label, Stack, useRootTheme} from '@sanity/ui'
+import {CheckmarkIcon, DesktopIcon, MoonIcon, SunIcon} from '@sanity/icons'
+import styled from 'styled-components'
+import {useColorScheme} from '../../../colorScheme'
+import {StudioTheme} from '../../../../theme'
+
+const StyledButton = styled(Button)`
+  cursor: pointer;
+`
+
+export function Appearance() {
+  const {scheme, setScheme, clearStoredScheme, usingSystemScheme} = useColorScheme()
+  const theme = useRootTheme().theme as StudioTheme
+
+  if (!theme?.__legacy)
+    return (
+      <Stack as="ul" space={[1, 2]}>
+        <Box padding={2}>
+          <Label size={1} muted>
+            Appearance
+          </Label>
+        </Box>
+
+        <Stack as="li" key="System">
```

Comment:
Is `key` necessary here? 

minor: I also wonder if using `<Grid>` over `<Stack>` is more appropriate here as it may be a better signifier of intent. Seeing a `<Stack>` component makes me think that this _could_ contain multiple buttons within, which would never be the case.